program/process/OpenHW Group IP Core - RTL Freeze Checklist and Release Process.md

```diff
@@ -0,0 +1,130 @@
+# OpenHW Group IP Core - RTL Freeze Checklist and Release Process
+
+
+This document describes the release process used by OpenHW Group for IP cores projects.
+
+In this process, OpenHW validates that a set of RTL Freeze checklist tasks have been completed prior to release.
+Checklist completion enables the creation and publishing of one or more "tags" on the project repositories. A consumer of the release accesses the tagged code associated with the release via the tags. OpenHW publishes the release name and description along with the release tags.
+
+
+The following aspects of the release process are described in this document: 
+* Naming the RTL Freeze release
+* Agree on RTL Freeze checklists, contents, and signoff criteria for a project
+* Open a Github issue for the checklist signoff
+* Complete (signoff) of all checklists associated
+* Create a Github tag on each repo included in the release
+* Close Github issue
+* Publish the release description and the Github tag(s)
+
+
+
+
+
+
+## Naming the release
+
+The RTL Freeze release is named with an alphanumberic string incorporating the version number. This name is used in release documentation, tag creation, checklist subdirectory name, and user documentation.
+For example, the first RTL Freeze for cv32e40p is named  cv32e40p_v1.0.0. 

```

Comment:
you may want to add that this is based on https://semver.org/ as discussed in https://github.com/openhwgroup/core-v-docs/issues/261

Reply:
I think the essence of #261 should (possibly)? live in this process document. For discussion. For the moment, I will add the reference to https://semver.org/

Reply:
The assets are in cv32e40p, verif, and docs. (In near future, docs will be integrated into cv32e40p and verif as appropriate. ) Because verif and docs cover multiple cores, i recommend the tag on those repos include "cv32e40p_v1.0.0" as part of the tag label string.